tools/docker/docker-compose-dev-redis-tls.yaml

```diff
@@ -32,6 +32,9 @@ x-environment:
   - EDA_PGSSLCERT=${EDA_PGSSLCERT:-./postgres_ssl_config/certs/server.crt}
   - EDA_PGSSLKEY=${EDA_PGSSLKEY:-./postgres_ssl_config/certs/server.key}
   - EDA_PGSSLROOTCERT=${EDA_PGSSLROOTCERT:-./postgres_ssl_config/certs/ca.crt}
+  - EDA_ANSIBLE_RULEBOOK_FLUSH_AFTER: ${EDA_ANSIBLE_RULEBOOK_FLUSH_AFTER:-100}
+  - EDA_DEFAULT_QUEUE_TIMEOUT: ${EDA_DEFAULT_QUEUE_TIMEOUT:-300}
```

Comment:
@jshimkus-rh Is this related to the logging issue?

Reply:
The FLUSH_AFTER definitely as it provides a way to manipulate the value for performance characterization.

The DEFAULT_QUEUE_TIMEOUT a bit less directly but being able to easily manipulate timeouts was valuable in preventing the system from going belly up while investigating the origins.

Reply:
@mkanoor is you question solved? Can we merge?